apps/prairielearn/src/pages/administratorInstitutions/administratorInstitutions.html.ts

```diff
@@ -89,3 +101,112 @@ export function AdministratorInstitutions({
     </html>
   `.toString();
 }
+
+function AddInstitutionModal({
+  csrf_token,
+  availableTimezones,
+}: {
+  csrf_token: string;
+  availableTimezones: Map<string, Timezone>;
+}) {
+  return html`
```

Comment:
We have a little "component" that can handle a lot of the modal boilerplate if you want to consider using that instead: https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/src/components/Modal.html.ts.

Here's an example:

https://github.com/PrairieLearn/PrairieLearn/blob/4f7b9a7fd6bec483511523157a67fe1980f76904/apps/prairielearn/src/pages/administratorFeatures/administratorFeatures.html.ts#L234

Note that splitting the body into another component isn't necessary; the only reason that page does is because we sometimes need to render the body independently of the modal as a whole.

Reply:
Converted to use the "component" and moved it into the body in commit [ed4f9f4](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/ed4f9f44057278fce12a5e2de1948446144ae6c6).